coreneuron/io/output_spikes.cpp

```diff
@@ -43,8 +43,8 @@ THE POSSIBILITY OF SUCH DAMAGE.
 #include "coreneuron/mpi/nrnmpidec.h"
 #include "coreneuron/utils/string_utils.h"
 #ifdef ENABLE_SONATA_REPORTS
-#include "reportinglib/records.h"
-#endif
+#include "bbp/sonata/reports.h"
```

Comment:
just curious : is there `bbp` in the include path? surprised by this!

Reply:
Yes, libsonata directory structure for include is the following:
include/bbp/sonata/*.h